src/pages/api/services/index.ts

```diff
@@ -179,9 +183,10 @@ const addOrUpdate =
       });
     } catch (e: any) {
       let message = e.toString();
-
       if (e?.response?.data?.message === 'Reference already exists') {
         message = `A branch with this name already exists on ${body?.destination}`;
+      } else {
+        console.error(e);
```

Comment:
> Is there no logging facility in this codebase? Using `console.error` seems more like a debugging instruction 🙂

Reply:
There is indeed none. I didn't feel I needed one.